testProject/build.gradle

```diff
@@ -32,26 +32,21 @@ sourceSets {
   }
 }
 
-def protobufDep = 'com.google.protobuf:protobuf-java:3.0.0-alpha-3'
-def protobufNanoDep = 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-3'
-
 dependencies {
-  compile protobufDep
+  compile 'com.google.protobuf:protobuf-java:3.0.0-alpha-3'
```

Comment:
The pre-defined variables makes it easier and less error-prone to upgrade the versions. What's the reason for this change?